tasks/Windows.yml

```diff
@@ -1,6 +1,11 @@
 ---
+- name: Ensure Chocolatey is installed
```

Comment:
I don't think this is needed. According to the doc

> If Chocolatey is missing from the system, the module will install it.

Reply:
There is an issue with the step hanging so I wanted to separate the 2 parts. I agree it should not be necessary. 
Though it does have the added benefit of being more self-documenting this way